lib/index.js

```diff
@@ -1,95 +1,50 @@
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-const bemNaming = require('bem-naming');
 const Readable = require('stream').Readable;
-const schemes = require('./schemes');
-
+const async = require('async');
+
+const walkers = require('./walkers');
+
+/**
+ * Scans levels in file system.
+ *
+ * If file or directory is valid BEM entity then will be called `add` with info about this file.
+ *
+ * @param {string[]} levels                The paths to levels.
+ * @param {object} options                 The options.
+ * @param {object} options.levels          The level map. The key is path to level, the value is options for this level.
```

Comment:
The first link: `value is OPTIONS` (means string or constant, or whatever), not `value is options`.
The second link is a personal blog, and word `options` is highlighted again.